dronegen/misc.go

```diff
@@ -14,9 +14,185 @@
 
 package main
 
+import (
+	"fmt"
+	"strings"
+)
+
 func promoteBuildPipeline() pipeline {
 	// TODO: migrate
-	return pipeline{}
+
+	aptPipeline := promoteAptPipeline()
+	return aptPipeline
+}
+
+// This function calls the build-apt-repos tool which handles the APT portion of RFD 0058.
+func promoteAptPipeline() pipeline {
+	testVersion := "6.2.31"
+	aptVolumeName := "aptrepo"
+	artifactPath := "/go/artifacts"
+
+	p := newKubePipeline("publish-apt-new-repos")
+	// p.Trigger = triggerPromote
+	p.Trigger = trigger{
+		Event:  triggerRef{Include: []string{"push"}},
+		Branch: triggerRef{Include: []string{"rfd/0058-package-distribution"}},
+	}
+	p.Workspace = workspace{Path: "/go"}
+	p.Volumes = []volume{
+		{
+			Name: aptVolumeName,
+			Claim: &volumeClaim{
+				Name: "drone-s3-aptrepo-pvc",
+			},
+		},
+		volumeTmpfs,
+	}
+	p.Steps = []step{
+		// {
```

Comment:
Can this commented-out stuff be deleted?

Reply:
Uncommented and tested (resolved)